apis/swagger.yml

```diff
@@ -1591,6 +1591,22 @@ definitions:
         type: "integer"
       Config:
         $ref: "#/definitions/ContainerConfig"
+      Architecture:
+        type: "string"
+      Os:
+        type: "string"
+      RootFS:
```

Comment:
I found that in moby's API, rootfs in ImageInspect cannot be a pointer, maybe we can a `x-nullable: false` to RootFS.

See:
``` go
// ImageInspect contains response of Remote API:
// GET "/images/{name:.*}/json"
type ImageInspect struct {
	ID              string `json:"Id"`
	RepoTags        []string
	RepoDigests     []string
	Parent          string
	Comment         string
	Created         string
	Container       string
	ContainerConfig *container.Config
	DockerVersion   string
	Author          string
	Config          *container.Config
	Architecture    string
	Os              string
	Size            int64
	VirtualSize     int64
	GraphDriver     GraphDriverData
	RootFS          RootFS
}
```